container-host-files/opt/scf/bin/klog.sh

```diff
@@ -5,192 +5,82 @@ set -e
 KLOG=${HOME}/klog
 
 if [ "$1" == "-h" ]; then
-    cat <<EOF
+	cat <<EOF
```

Comment:
Unnecessary change here and below. Also, you may want to set up your IDE for spaces rather than tabs.

Reply:
I guess indentation is not standardized in the file, therefore even spaces won't work. For now, will revert indentation related changes and handle them in a separate commit.